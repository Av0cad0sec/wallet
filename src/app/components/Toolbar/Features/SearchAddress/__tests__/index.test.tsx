import * as React from 'react'
import { render } from '@testing-library/react'

import { SearchAddress } from '..'

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    }
  },
}))

describe('<SearchAddress  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<SearchAddress />)
    expect(loadingIndicator.container.firstChild).toMatchSnapshot()
  })
})
