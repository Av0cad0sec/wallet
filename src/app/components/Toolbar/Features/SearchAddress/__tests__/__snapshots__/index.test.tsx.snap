// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchAddress  /> should match snapshot 1`] = `
.c5 {
  display: inline-block;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  width: 24px;
  height: 24px;
  fill: #666666;
  stroke: #666666;
}

.c5 g {
  fill: inherit;
  stroke: inherit;
}

.c5 *:not([stroke])[fill="none"] {
  stroke-width: 0;
}

.c5 *[stroke*="#"],
.c5 *[STROKE*="#"] {
  stroke: inherit;
  fill: none;
}

.c5 *[fill-rule],
.c5 *[FILL-RULE],
.c5 *[fill*="#"],
.c5 *[FILL*="#"] {
  fill: inherit;
  stroke: none;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: solid 2px #DADADA;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  max-width: 600px;
  height: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  border-radius: 30px;
}

.c3 {
  margin: 36px;
  font-size: 18px;
  line-height: 24px;
  color: #DADADA;
}

.c6 {
  box-sizing: border-box;
  font-size: inherit;
  font-family: inherit;
  border: none;
  -webkit-appearance: none;
  background: transparent;
  color: inherit;
  width: 100%;
  padding: 11px;
  font-weight: 600;
  margin: 0;
  border: 1px solid rgba(0,0,0,0.33);
  border-radius: 4px;
  outline: none;
  border: none;
  padding-left: 48px;
}

.c6:focus {
  outline: none;
  box-shadow: 0 0 2px 2px #6FFFB0;
}

.c6:focus > circle,
.c6:focus > ellipse,
.c6:focus > line,
.c6:focus > path,
.c6:focus > polygon,
.c6:focus > polyline,
.c6:focus > rect {
  outline: none;
  box-shadow: 0 0 2px 2px #6FFFB0;
}

.c6:focus::-moz-focus-inner {
  border: 0;
}

.c6::-webkit-input-placeholder {
  color: #AAAAAA;
}

.c6::-moz-placeholder {
  color: #AAAAAA;
}

.c6:-ms-input-placeholder {
  color: #AAAAAA;
}

.c6::-webkit-search-decoration {
  -webkit-appearance: none;
}

.c6::-moz-focus-inner {
  border: none;
  outline: none;
}

.c6:-moz-placeholder,
.c6::-moz-placeholder {
  opacity: 1;
}

.c1 {
  position: relative;
  width: 100%;
}

.c2 {
  position: absolute;
  left: 11px;
  top: 50%;
  -webkit-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  pointer-events: none;
}

.c4 {
  position: absolute;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-packjustify: center;
  -webkit-justify: center;
  -ms-flex-packjustify: center;
  justify: center;
  top: 50%;
  -webkit-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  pointer-events: none;
  left: 12px;
}

@media only screen and (max-width:768px) {
  .c0 {
    border: solid 2px #DADADA;
  }
}

<form>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <span
          class="c3"
        >
          toolbar.search.placeholder
        </span>
      </div>
      <div
        class="c4"
      >
        <svg
          aria-label="Search"
          class="c5"
          viewBox="0 0 24 24"
        >
          <path
            d="M15,15 L22,22 L15,15 Z M9.5,17 C13.6421356,17 17,13.6421356 17,9.5 C17,5.35786438 13.6421356,2 9.5,2 C5.35786438,2 2,5.35786438 2,9.5 C2,13.6421356 5.35786438,17 9.5,17 Z"
            fill="none"
            stroke="#000"
            stroke-width="2"
          />
        </svg>
      </div>
      <input
        autocomplete="off"
        class="c6"
        data-testid="searchaddress"
        value=""
      />
    </div>
    
  </div>
</form>
`;
